src/cmd/package.go

```diff
@@ -68,13 +75,23 @@ func choosePackage(args []string) string {
 	return path
 }
 
+func cachePathClean(cachePath string) bool {
+	var isCleanPath = regexp.MustCompile(`^[a-zA-Z0-9\_\-\/\.\~]+$`).MatchString
+	if !isCleanPath(cachePath) {
+		message.Warn(fmt.Sprintf("Invalid characters in Zarf cache path, defaulting to ~/%s", config.ZarfDefaultImageCachePath))
+		return false
+	}
+	return true
+}
+
 func init() {
 	rootCmd.AddCommand(packageCmd)
 	packageCmd.AddCommand(packageCreateCmd)
 	packageCmd.AddCommand(packageDeployCmd)
 	packageCmd.AddCommand(packageInspectCmd)
 
 	packageCreateCmd.Flags().BoolVar(&config.DeployOptions.Confirm, "confirm", false, "Confirm package creation without prompting")
+	packageCreateCmd.Flags().StringVar(&zarfImageCache, "zarf-cache", config.ZarfDefaultImageCachePath, "Specify the location of the Zarf image cache")
```

Comment:
Not necessary for this PR, but this is another example of where [Viper](https://github.com/spf13/viper) might actually be useful as this is flag you'd likely want to be durable and not have to type it each time.